tools/mqppep/mqppep_preproc.xml

```diff
@@ -0,0 +1,520 @@
+<tool id="mqppep_preproc" name="MaxQuant Phosphopeptide Preprocessing" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" python_template_version="3.5">
+    <description>
+        Transform the output of MaxQuant for phosphoproteome-enriched samples to prepare it for statistical anlaysis.
+    </description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements"/>
+    <command detect_errors="exit_code"><![CDATA[
+      (
+        echo '--- localization-filter step:'
+      ) && (
+        \$CONDA_PREFIX/bin/Rscript '$__tool_directory__/MaxQuantProcessingScript.R'
+        -i '$phosphoSites'
+        #if $pst_py_selector == "y"
+            --enriched Y
+        #else
+            --enriched ST
+        #end if
+        --phosphoCol '$phosphocol_script'
+        --startCol '$startcol_script'
+        --intervalCol $intervalCol
+        --localProbCutoff $localProbCutoff
+        --collapse_func $collapseFunc
+        -o '$phosphoPepIntensities'
+        --locProbCutoffGraph $locProbCutoffGraph
+        --enrichGraph $enrichGraph
+        --locProbCutoffGraph_svg $locProbCutoffGraph_svg
+        --enrichGraph_svg $enrichGraph_svg
+        --filtered_data $filteredData_tabular
+        --quant_data $quantData_tabular
+      ) && (
+        echo '... end localization-filter.'
+      ) && (
+        echo '--- kinase-mapping step:'
+      ) && (
+        \$CONDA_PREFIX/bin/perl '$__tool_directory__/PhosphoPeptide_Upstream_Kinase_Mapping.pl'
```

Comment:
see above, we should not rely on CONDA_PREFIX if those tools are run in Docker, this PREFIX will not exists.

Reply:
@bgruening My tool **does not run** in our production Galaxy without the `CONDA_PREFIX`.

My tool **does run** with `planemo test --biocontainers .`

Reply:
Implemented in commit 3d51a38113ad440ac4b8e6e569d9f6cae936cf7a